network/p2p/scoring/registry_test.go

```diff
@@ -577,6 +620,82 @@ func TestPeerSpamPenaltyClusterPrefixed(t *testing.T) {
 	}
 }
 
+// TestScoringRegistrySilencePeriod ensures that the scoring registry does not penalize nodes during the silence period, and
+// starts to penalize nodes only after the silence period is over.
+func TestScoringRegistrySilencePeriod(t *testing.T) {
+	peerID := unittest.PeerIdFixture(t)
+	silenceDuration := 5 * time.Second
+	silencedNotificationLogs := atomic.NewInt32(0)
+	hook := zerolog.HookFunc(func(e *zerolog.Event, level zerolog.Level, message string) {
+		if level == zerolog.DebugLevel {
+			if message == "ignoring invalid control message notification for peer during silence period" {
```

Comment:
Please consider casting this message as a constant that is shared between the code and test logic. In this way changing the log wording doesn't break the test.

Reply:
https://github.com/onflow/flow-go/pull/5084/commits/6d9f1d2ad6f61c25bae45946e0bb84b291cadd58